style.css

```diff
@@ -494,7 +494,6 @@ hr {
 .header_nav_desktop_version {
   display: flex;
   flex-direction: row;
-  align-items: center;
   justify-content: center;
   padding: 0;
   padding-top: 1em;
```

Comment:
As previously requested, it is important to make the design much similar to that of the Figma design in order to be consistent with the template.

Address the following:
- The height of the nav menu (instead of using min-height: 10vh; change it to 100vh). This height is making the image that the previous reviewer spoke about to be visible.
- the close button should be properly positioned 
- the spacing between the nav links
- pls addres the nav link color to `color: #3A4A42` as previously asked as it is still `color: #344563;`

Your Design

![harmburger](https://user-images.githubusercontent.com/84930355/224071389-4ee93cba-3150-4bf6-b3b0-d881e213e7d4.PNG)

Figma design

![figHArm](https://user-images.githubusercontent.com/84930355/224074106-c56e9b78-621c-46e9-9954-f94ff4c06cfe.PNG)